client/js/components/user/DpMasterToebList.vue

```diff
@@ -593,6 +532,67 @@ export default {
       } else if (this.editModeElementId === 0) {
         dplan.notify.error(Translator.trans('error.api.generic'))
       }
+    },
+
+    updateOrga (ident, updatedField, status) {
+      this.updatedItems = this.updatedItems.map(item => {
+        let newItem = item
+        if (item.ident === ident) {
+          newItem = { ...newItem, ...updatedField }
+        }
+        return newItem
+      })
+
+      this.rowItems = this.rowItems.map(item => {
+        let newItem = item
+        if (item.ident === ident) {
+          newItem = { ...newItem, ...updatedField }
+        }
+        return newItem
+      })
```

Comment:
this function is nearly the same as the one above, have you considered extracting this in its own function and use the return of the new function on both instances?

Reply:
I extracted it in a new function `updateItems` in [c706ee7](https://github.com/demos-europe/demosplan-core/pull/2621/commits/c706ee785dffafea8563a6127954102775662818)